src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -372,6 +373,20 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
       });
   }
 
+  private checkNewCountryEquality(newCountry: SkyCountryFieldCountry) {
+    if (this._selectedCountry && newCountry) {
```

Comment:
You can write this more succinctly without all the nested `if`s:

```
if (country1 && country2) {
  return country1.iso2 === country2.iso2;
}

return country1 === country2;
```

Reply:
Done. However, I did have to make a slight tweak to the last line which I outline the reasoning for in a code comment at that line.